GsaGH/Components/0_Model/GetModelGeometry.cs

```diff
@@ -269,7 +270,7 @@ private enum FoldMode {
       Params.Output[i].Name = "3D Members in [" + unitAbbreviation + "]";
 
       i = 1;
-      for (int j = 1; j < 7; j++) {
+      for (int j = 1; j < 8; j++) {
```

Comment:
what is 8? can you change this magic number?

Reply:
that is the number of outputs - 1?!